src/tags/object/TimeSeries.js

```diff
@@ -191,7 +191,7 @@ const Model = types
           current = self.timeformat ? self.parseTime(value) : value;
           timestamps[i] = current;
 
-          if (current < previous) {
+          if (!previous && current < previous) {
```

Comment:
How will this check work for positive axis with this change?

Assuming the time values of `[1, 4, 2, 3, 5, 6]` 
This check will not catch the out of order value `4`, `2` as previous is never falsey.

The use of undefined here with a check for falsey instead of strict checks will not allow this to work for most other cases.

What is the specific case you are looking to solve for with regards to negative values?  If we could add those to the [timeseries tests](./e2e/tests/timeseries.test.js) and have them pass it would help define the case and ensure no regressions.